src/mpid/ch4/src/ch4_init.c

```diff
@@ -435,6 +435,14 @@ int MPID_Init(int *argc, char ***argv, int requested, int *provided)
         if (mpi_errno != MPI_SUCCESS) {
             MPIR_ERR_POPFATAL(mpi_errno);
         }
+#ifndef MPIDI_CH4_DIRECT_NETMOD
+        mpi_errno = MPIDI_vci_pool_alloc(n_vsis_provided, n_vnis_provided);
```

Comment:
Where are the values for `n_vsis_provided` and `n_vnis_provided` assigned?

Reply:
They are assigned inside the `netmod` and `shmmod` init hook.